Procfile

```diff
@@ -0,0 +1,2 @@
+release: npm run db:migrate
+web: npm start
```

Comment:
Interesting – I had configured Heroku to run our app without a procfile and deleted it a little while ago.  

How/when is `release` triggered?  It _would_ be nice to have Heroku run migrations automatically... I wonder if there's a way to configure this _without_ the Heroku-only procfile?  Not opposed to reinstating it, just curious.

Reply:
I also wanted to avoid `Procfile`, but I could not find documentation or a hook in https://github.com/heroku/heroku-buildpack-nodejs for running a command on release any other way 😢